src/select/__tests__/select-highlight.scenario.js

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+
+import {Select} from '../index.js';
+
+export const name = 'select-highlight';
+
+export const component = () => {
+  const [value, setValue] = React.useState([]);
+  return (
```

Comment:
I think it would be the best if the dropdown would be opened for this test case - you can learn how to do that here https://screener.io/v2/docs/test-interactions

Reply:
done